install/kubernetes/helm/istio/values-istio-multicluster-gateways.yaml

```diff
@@ -24,3 +24,40 @@ gateways:
     env:
       # Needed to route traffic via egress gateway if desired.
       ISTIO_META_REQUESTED_NETWORK_VIEW: "external"
+  istio-ingressgateway:
+    ports:
+      # This is the port where sni routing happens
+      # Note that for the multicluster multigateway example this port
+      # needs to be the first in the list unless AWS ELB uses port 80 for
+      # health checking and marks it an unhealthy if no gateway is
+      # configured on port 80.
+      # https://github.com/istio/istio/issues/12503
+    - port: 15443
+      targetPort: 15443
+      name: tls
+      ## You can add custom gateway ports
+    - port: 80
```

Comment:
Is it possible to fix this issue without having to copy all of the other ports as well? It would be nice to not have this duplication as it is hard to keep all the values files in sync when things change.

Reply:
dont think so with helm template... however, if we fix this in the default config, the prob went away :)